airbyte-ci/connectors/pipelines/pipelines/dagger/containers/git.py

```diff
@@ -19,8 +24,12 @@ async def checked_out_git_container(
     We fetch the diffed branch from the origin remote and the current branch from the target remote.
     We then checkout the current branch.
     """
+    origin_repo_url = AIRBYTE_REPO_URL
     current_git_branch = current_git_branch.removeprefix("origin/")
     diffed_branch = current_git_branch if diffed_branch is None else diffed_branch.removeprefix("origin/")
+    if github_token := os.environ.get("CI_GITHUB_ACCESS_TOKEN"):
+        origin_repo_url = get_authenticated_repo_url(origin_repo_url, github_token)
+        target_repo_url = get_authenticated_repo_url(repo_url, github_token)
```

Comment:
I don't think this should block this PR but you might want to consider mounting the token as a secret in the dagger container.

Reply:
Absolutely! I made the change.